src/main/java/hw01/Human.java

```diff
@@ -1,4 +1,127 @@
 package hw01;
 
-public class Human {
+public class Human implements HumanActions {
+    private String name;
+    private String surname;
+    private Integer dateOfBirthYear;
+    private Integer IQ;
```

Comment:
pls change field naming to lowerCase, full uppercase used for constants
[https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](url)